README.md

```diff
@@ -253,6 +253,18 @@ make sure you have added your key as an admin user, or you’re using `anon-acce
 admin-access` in the configuration.
 
 ### A note about RSA keys
```

Comment:
I think this line should be removed

Reply:
Not this one 😆  I meant `### A note about RSA keys`, I should've been more explicit. And while you're at it, could you squash all the commits into one commit?

Reply:
Ah lol! My bad I didn't read that properly haha. And yeah can do!

Reply:
Done!